tabs/src/Home.tsx

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import FeedComponent from './components/FeedComponent';
+import RewardsComponent from "./components/RewardsComponent";
 
 const Home: React.FC = () => {
   return (
     <div>
-      <FeedComponent />
+            <RewardsComponent />
```

Comment:
This should go on a new `src/Rewards.tsx` page.

Reply:
And add the Rewards page to the FooterComponent.

Reply:
> This should go on a new `src/Rewards.tsx` page.

resolved